Library/Formula/sniper.rb

```diff
@@ -0,0 +1,7 @@
+class Sniper < Formula
+  desc "osx command line app terminator"
+  homepage "https://github.com/toshi0383/sniper"
+  url "https://github.com/toshi0383/sniper.git", :tag => "1.0.0", :revision => "8a655740145b0f9846b02eadfd491b08ef287e57"
+  head "https://github.com/toshi0383/sniper.git"
+
```

Comment:
You're missing an install and test block; take a look at the Formula Cookbook: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md